src/bentoml_cli/deployment.py

```diff
@@ -58,6 +57,13 @@ def shared_decorator(f: t.Callable[..., t.Any]):
             f = opt(f)
         return f
 
+    def shared_decorator(f: t.Callable[..., t.Any]) -> t.Callable[..., t.Any]:
+        return click.argument(
+            "deployment-name",
+            type=click.STRING,
+            required=True,
+        )(shared_options_decorator(f))
```

Comment:
You can just put the click.argument in the list above right?

Reply:
In `get`, `delete` and `terminate`, `deployment-name` is a mandatory `click.argument`, but it's an optional `click.option` in `update`, since we can also use a json file.

Reply:
then just pass that argument via the first decorator, no need to do this